ibis/backends/dask/execution/aggregations.py

```diff
@@ -77,7 +77,7 @@ def execute_aggregation_dataframe(
     pieces = []
     for metric in op.metrics:
         piece = execute(metric, scope=scope, timecontext=timecontext, **kwargs)
-        piece = maybe_wrap_scalar(piece, metric)
+        piece = coerce_to_output(piece, metric)
```

Comment:
is this a consistent name we use in other backends?

Reply:
Yep - I'm emulating the pandas code https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/execution/generic.py#L437